pydoof/search_api/search.py

```diff
@@ -81,54 +81,32 @@ def query(hashid, query, filter_=None, exclude=None, index_name=None,
     )
 
 
-def suggest(hashid, query, filter_=None, exclude=None, sort=None, page=None,
-            rpp=None, transformer=None, no_stats=None, **opts):
+from typing import List
```

Comment:
But... put this at top, don't commit the suggestion as it is please :) at least the one I did before (that was not a simpler-one)

Reply:
Ok, I do this on next commit. Thanks!